paper_figures/code/maxsavings_boxplot.py

```diff
@@ -27,11 +27,13 @@
 mef_savings_sweep = np.zeros((len(regions), len(month_arr)))
```

Comment:
List is fine but we can also use ragged arrays if we want to stay within numpy.

Reply:
I left this as is, but @emusabandesu feel free to change the data structure to whatever is easiest for plotting (I didn't test the plotting code works well, but just that the tariffs are processed in a loop)